watchDistFile.ts

```diff
@@ -0,0 +1,5 @@
+// var watch = require('node-watch');
```

Comment:
WIP. I want to keep it here for a reminder to fix the watching of the `dist` directory.